build/linux/installer/scripts/livenessprobe.sh

```diff
@@ -17,18 +17,15 @@ syslogSetup() {
 }
 
 if [[ "${CONTROLLER_TYPE}" == "DaemonSet" ]]; then
-  if [[ "${CONTAINER_TYPE}" == "PrometheusSidecar" && "${GENEVA_LOGS_INTEGRATION}" == "true" && -d "/var/run/mdsd-ci" ]]; then
-    syslogSetup
-  else
-    syslogSetup
-    CURRENT_LOGS_AND_EVENTS_ONLY=${LOGS_AND_EVENTS_ONLY}
-    ruby /opt/dcr-config-parser.rb > /dev/write-to-traces 2>&1
-    source /opt/dcr_env_var
-    if [ "${LOGS_AND_EVENTS_ONLY}" != "${CURRENT_LOGS_AND_EVENTS_ONLY}" ]; then
-      echo "dcr_env_var has been updated - dcr config changed" > /dev/termination-log
-      exit 1
-    fi
-  fi
+  syslogSetup
```

Comment:
could you share detail more about always runs after syslogSetup when CONTROLLER_TYPE == DaemonSet.